src/interfaces/InjectedSenderWallet.ts

```diff
@@ -101,16 +91,15 @@ export interface RequestSignTransactionsParams {
 }
 
 interface InjectedSenderWallet {
-  init: (params: InitParams) => Promise<InitResponse>;
+  isSender: string,
   getAccountId: () => string;
   getRpc: () => Promise<GetRpcResponse>;
   requestSignIn: (
     params: RequestSignInParams
   ) => Promise<RequestSignInResponse>;
-  signOut: () => Promise<SignOutResponse>;
+  signOut: () => SignOutResponse;
   isSignedIn: () => boolean;
-  onAccountChanged: (callback: AccountChangedCallback) => void;
-  onRpcChanged: (callback: RpcChangedCallback) => void;
+  on: (event: string, callback: Callback) => void;
   // TODO: Determine return type.
   sendMoney: (params: SendMoneyParams) => Promise<unknown>;
   signAndSendTransaction: (
```

Comment:
It looks like the multi-transaction method, `requestSignTransactions` has been renamed to `signAndSendTransactions` when testing the extension directly in my browser. This contradicts the docs: https://docs.senderwallet.io/api-reference/near-provider-api#near.requestsigntransactions

Reply:
That's true. The next release should fix the the API inconsistency issue, which should be ready earlier but it's pending on review by Chrome Web Store.